src/analyze/analyze.c

```diff
@@ -1136,28 +1153,37 @@ static int graph_one_property(sd_bus *bus, const UnitInfo *u, const char* prop,
 static int graph_one(sd_bus *bus, const UnitInfo *u, char *patterns[], char *from_patterns[], char *to_patterns[]) {
         int r;
 
+        typedef struct ArgDotDep {
+                enum dot arg_dot;
+                UnitDependency dependency;
+        } ArgDotDep;
+
+        static const ArgDotDep arg_dot_deps[] = {
```

Comment:
no need to name the structure type if we never need it again. use this:

```c
static const struct {
        DotDependencyKind kind;
        UnitDependency dependency;
} dependency_table[] = {
        …
};
```

Reply:
done